tools/autograd/gen_python_functions.py

```diff
@@ -15,6 +15,12 @@
     from tools.shared.module_loader import import_module
     CodeTemplate = import_module('code_template', 'aten/src/ATen/code_template.py').CodeTemplate
 
+try:
+    from src.ATen.tensor_options_utils import *
+except ImportError:
+    from tools.shared.module_loader import import_module
+    TOUtils = import_module('tensor_options_utils', 'aten/src/ATen/tensor_options_utils.py')
```

Comment:
Just curious, why is the try/catch setup necessary? Noticed the earlier use prior to this diff and was hoping there was some way to get rid of it :)